Assignment2 crud operation

```diff
@@ -0,0 +1,164 @@
+//Kalpvriksha Program | Batch-2025: C Programming Assignments covering CRUD operations.
+
+//assginment 2: CRUD operation, by creating a file named users.txt
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct
+{
+    int id;
+    char name[50];
+    int age;
+} User;
+
+void addUser();
+void displayUsers();
+void updateUser();
+void deleteUser();
+
+int main()
+{
+    int choice;
+    do
+    {
+        printf("1. Add User\n2. Display Users\n3. Update User\n4. Delete User\n5. Exit\n");
+        printf("Enter your choice: ");
+        scanf("%d", &choice);
+
+        switch (choice)
+        {
+        case 1:
+            addUser();
+            break;
+        case 2:
+            displayUsers();
+            break;
+        case 3:
+            updateUser();
```

Comment:
can you handle scenario, where user should not add data if user ID already present?

right now, if there is 2 same user ID's, code is updating for two users and deleting 2 users.
Ex :

2,Abhi,10
2,sameer,20